modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You installed the OpenShift CLI (`oc`).
+* Your `oc` client must be updated to at least verion 4.13.0. 
+* Your {product-title} cluster must be installed on either the AWS or Azure platform. 
++
+For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_.
+
+
+.Procedure
+. Verify that the `RetrievedUpdates` condition is `True` in the Cluster Version Operator (CVO) by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion/version -o=jsonpath="{.status.conditions[?(.type=='RetrievedUpdates')].status}"
+----
++ 
+For more information about cluster version condition types, see _Understanding cluster version condition types_.
+
+. If the condition `RetrievedUpdates` is `False`, change the channel to `stable-<4.y>` or `fast-<4.y>` with the following command: 
++
+[source,terminal]
+----
+$ oc adm upgrade channel <channel>
+----
++
+After setting the channel, verify if `RetrievedUpdates` is `True`. 
++
+For more information about channels, see _Understanding update channels and releases_.
+
+. Migrate to the mult-architecture payload with following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-multi-arch
+----
+
+.Verification 
+
+* After you execute the upgrade command, your cluster now uses the multi-arch payload, but the cluster operators continue to upgrade in the background. You can monitor the upgrade process by running the following command: 
++
+[source,terminal]
+----
+$ watch  'oc get co; oc get mcp; oc adm upgrade'
+----
++
+For `oc get co`, expect `AVAILABLE=True`, `PROGRESSING=False`, and `DEGRADED=False` on all cluster Operators. 
```

Comment:
I'm still not excited about recommending `Progressing=False` here, with more details [in this thread][1].

[1]: https://github.com/openshift/openshift-docs/pull/58104#discussion_r1161986216

Reply:
Hmm, if `Progressing=False` is confusing, should we drop the `Progressing=False` or drop [all these lines](https://github.com/openshift/openshift-docs/pull/58104/files#diff-04cec016f67c8e6e057bafbb9eaed00f01da14e4a8e7de1c6c19eaa8889cc019R59-R68)? @wking do you have any suggestion?

Reply:
For now, I've commented out that section and replaced it with an important note for users to consider deploying machine health checks to notice and recover when machines fail to launch. :) Are we ok for now until https://issues.redhat.com/browse/OCPBUGS-8256 is resolved? @wking @shellyyang1989 @LalatenduMohanty

Reply:
Here is my suggestion only mention 
```
$ watch  'oc get co; oc get mcp; oc adm upgrade'

For `oc get co`, expect `AVAILABLE=True`, `PROGRESSING=False`, and `DEGRADED=False` on all cluster Operators. 

You can also use `oc get mcp` and `oc adm upgrade` to get some more details about the upgrade.
```